feature-libs/asm/components/customer-emulation/customer-emulation.component.ts

```diff
@@ -25,12 +45,54 @@ export class CustomerEmulationComponent implements OnInit, OnDestroy {
     );
     this.isCustomerEmulationSessionInProgress$ =
       this.asmComponentService.isCustomerEmulationSessionInProgress();
+
+    this.cartId.valueChanges.subscribe((response: string) => {
+      this.cartIdExists = response.trim().length > 0;
+    });
+
+    this.activeCartFacade.getActiveCartId().subscribe((response) => {
+      if (response) {
+        this.cartId.setValue(response);
+      }
+    });
```

Comment:
I would like to understand the purpose of the cart facade logic here.  Cart binding is done by the asm endpoint.  Let's discuss.

Reply:
I will recheck to see if this functionality is needed